src/applications/gi/components/ResponsiveFacilityMapTable.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
```

Comment:
should this be in the `src/applications/gi/components/profile` directory since it is specific to profile?

Reply:
Yeah I can move that, was going off of Dustin's original suggestion. It does make more since for it to be there.